pkg/wolfssl/Makefile

```diff
@@ -0,0 +1,21 @@
+PKG_NAME=wolfssl
+PKG_URL=https://github.com/wolfssl/wolfssl.git
+#PKG_VERSION=a79f9c93c98f1af4e6b2c70abe8208bab8919944
+PKG_VERSION=f7cd8a0f15774f8fdb686c13008ec48601b64b28
```

Comment:
Will update once PR to address old-style function defs is merged into wolfSSL master:
https://github.com/wolfSSL/wolfssl/pull/1029